packages/babel-plugin-strip-runtime/src/index.ts

```diff
@@ -53,17 +53,16 @@ export default declare<PluginPass>((api) => {
           }
 
           if (this.opts.extractStylesToDirectory && this.styleRules.length > 0) {
-            // Build filename of the css file
-            const currentFilename = basename(filename);
-            const cssFilename = `${currentFilename.substring(
-              0,
-              currentFilename.lastIndexOf('.')
-            )}.compiled.css`;
```

Comment:
Ah is there protection against accidental overwriting of an existing `.compiled.css` file?

For example, if we have `app+v1.ts` and `app=v1.ts`, would the generated `.compiled.css` file for the second file overwrite the generated `.compiled.css` file for the first file?